src/gvmd.c

```diff
@@ -2568,6 +2573,25 @@ gvmd (int argc, char** argv)
         }
       return EXIT_SUCCESS;
     }
+  
+  if (dump_vt_verification)
+    {
+      int ret;
+
+      proctitle_set ("gvmd: --dump-vt-verification");
+  
+      if (option_lock (&lockfile_checking))
+        return EXIT_FAILURE;
+
+      ret = manage_dump_vt_verification (log_config, &database);
+      log_config_free ();
+      if (ret)
+        {
+          printf ("Failed to rebuild SCAP data.\n");
```

Comment:
This message seems to not match what the code is actually doing.

Reply:
I've fixed this in ca30e385696bb33255883bad8d6c319d5b98d8dc